.changeset/beige-cherries-wink.md

```diff
@@ -0,0 +1,5 @@
+---
+"@soundxyz/sdk": patch
+---
+
+minters v2
```

Comment:
be descriptive, this is what goes on the release

Reply:
sounds good, will change